src/main/java/Command.java

```diff
@@ -0,0 +1,4 @@
+abstract public class Command {
+    abstract public void execute(TaskList taskList, Ui ui, Storage storage);
```

Comment:
Perhaps you can add some header comments for public class and its method.

Reply:
Added javadocs in later commits.